dev/TeachingTip/TestUI/TeachingTipPage.xaml.cs

```diff
@@ -312,68 +311,80 @@ public void OnSetTitleButtonClicked(object sender, RoutedEventArgs args)
             }
         }
 
-        public void OnSetSubtextButtonClicked(object sender, RoutedEventArgs args)
+        public void OnSetSubtitleButtonClicked(object sender, RoutedEventArgs args)
         {
-            if (this.SubtextComboBox.SelectedItem == SubtextNo)
+            if (this.SubtitleComboBox.SelectedItem == SubtitleNo)
             {
-                getTeachingTip().Subtext = "";
+                getTeachingTip().Subtitle = "";
             }
-            else if (this.SubtextComboBox.SelectedItem == SubtextSmall)
+            else if (this.SubtitleComboBox.SelectedItem == SubtitleSmall)
             {
-                getTeachingTip().Subtext = "Short Subtext.";
+                getTeachingTip().Subtitle = "Short Subtitle.";
             }
             else
             {
-                getTeachingTip().Subtext = "This is a much longer subtext that might cause some issues if we don't do the right thing..." +
-                    "This is a much longer subtext that might cause some issues if we don't do the right thing...";
+                getTeachingTip().Subtitle = "This is a much longer subtitle that might cause some issues if we don't do the right thing..." +
+                    "This is a much longer subtitle that might cause some issues if we don't do the right thing...";
             }
         }
 
-        public void OnSetActionButtonTextButtonClicked(object sender, RoutedEventArgs args)
+        public void OnSetActionButtonContentButtonClicked(object sender, RoutedEventArgs args)
         {
-            if (this.ActionButtonTextComboBox.SelectedItem == ActionButtonTextNo)
+            if (this.ActionButtonContentComboBox.SelectedItem == ActionButtonContentNo)
             {
-                getTeachingTip().ActionButtonText = "";
+                getTeachingTip().ActionButtonContent = "";
             }
-            else if (this.ActionButtonTextComboBox.SelectedItem == ActionButtonTextSmall)
+            else if (this.ActionButtonContentComboBox.SelectedItem == ActionButtonContentSmall)
             {
-                getTeachingTip().ActionButtonText = "A:Short Text.";
+                getTeachingTip().ActionButtonContent = "A:Short Text.";
+            }
+            else if (this.ActionButtonContentComboBox.SelectedItem == ActionButtonContentLong)
+            {
+                getTeachingTip().ActionButtonContent = "A:This is a much longer button text that might cause some issues if we don't do the right thing...";
             }
             else
             {
-                getTeachingTip().ActionButtonText = "A:This is a much longer subtext that might cause some issues if we don't do the right thing...";
+                var button = new Button();
```

Comment:
What's with the new fallbacks? Not just an API rename, but a new value?

Reply:
This is in the test page code, I'm testing putting a non string object in the content of the button, in this case, that content is another button.

---
In reply to: [265700838](https://github.com/Microsoft/microsoft-ui-xaml/pull/409#discussion_r265700838) [](ancestors = 265700838)